tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -1251,6 +1251,41 @@ SpirvInstruction *SpirvEmitter::loadIfAliasVarRef(const Expr *expr,
   return instr;
 }
 
+QualType SpirvEmitter::redeclSpecialVarType(QualType type,
```

Comment:
`redecl` is not a word or a common abbreviation. Also "redeclare special variable type" is so vague it is hard to know what it is doing at the call sights.  A better name might be `expandNoInterpolationParamToArray`. Personally, I do not have a problem with long names. Good IDEs can save you from typing it in all of the time.

Reply:
OK. Thanks.